src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -632,6 +643,52 @@ public Stream GetTempStream()
             }
         }
 
+        private class StreamTimeSeriesActions : StreamActionsBase, ITimeSeriesActions
+        {
+            private readonly DocumentsOperationContext _context;
+
+            public StreamTimeSeriesActions(BlittableJsonTextWriter writer, DocumentsOperationContext context, string propertyName) : base(writer, propertyName)
+            {
+                _context = context;
+            }
+
+            public unsafe void WriteTimeSeries(TimeSeriesItem item)
+            {
+                if (First == false)
+                    Writer.WriteComma();
+                First = false;
+
+                Writer.WriteStartObject();
+
+                Writer.WritePropertyName(nameof(TimeSeriesItem.DocId));
+                Writer.WriteString(item.DocId);
```

Comment:
In `StreamCounterActions` we are setting skipEscaping also here. What is the correct way?